no_drama/__main__.py

```diff
@@ -0,0 +1,179 @@
+#!/usr/bin/env python
+
+import argparse
+import tempfile
+import os
+import os.path
+import shutil
+import subprocess
+import zipfile
+import glob
+import fnmatch
+
+from string import Template
+from functools import partial
+
+
+this_file = os.path.realpath(__file__)
+this_directory = os.path.dirname(this_file)
+templates_dir = os.path.join(this_directory, 'templates')
+build_lib = os.path.join(this_directory, 'build_lib')
+build_files = os.path.join(this_directory, 'build_files')
+
+
+def save_wheels(destination, packages=None, requirements_file=None,
+                find_links=None):
+    call_list = ["pip", "wheel", "--wheel-dir=%s" % destination]
+
+    if packages is not None:
+        call_list += packages
+
+    if find_links is not None:
+        call_list += ['--find-links=%s' % find_links]
+
+    if requirements_file is not None:
+        call_list += ['-r', requirements_file]
+
+    subprocess.call(call_list)
+
+
+def get_file_from_dir(dir, filename, mode='r'):
+    full_path = os.path.join(dir, filename)
+    return open(full_path, mode)
+
+
+def get_template(filename, mode='r'):
+    template_file = get_file_from_dir(templates_dir, filename, mode=mode)
```

Comment:
I think it'd make more sense to `try`/`except` wherever the error should be appropriately handled.